contracts/interfaces/IVersion.sol

```diff
@@ -0,0 +1,13 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
```

Comment:
Set the version to `^0.8.19` instead of `=0.8.19`, so that any inheriting contracts can use any solidity version up to 0.9.0.